src/configuration.rs

```diff
@@ -3,7 +3,7 @@
 /// MQTT broker IPv4 address
 ///
 /// In the default configuration, the IP address is defined as 10.34.16.10.
-pub const MQTT_BROKER: [u8; 4] = [10, 34, 16, 10];
+pub const MQTT_BROKER: [u8; 4] = [10, 35, 16, 10];
```

Comment:
Spurious

```suggestion
pub const MQTT_BROKER: [u8; 4] = [10, 34, 16, 10];
```

Reply:
Woops, looks like my local cache slipped in.

What are your thoughts on adding `src/configuration.rs` to the ignore list? The downside is that you don't see the diff in `git diff` if you modify it

Reply:
I'd rather not. Ultimately the hardcoded configuration should go away: replaced by genericism (or just runtime configuration with sufficiently large upper limit of 32 sample static buffers) for those the first two parameters and runtime configuration by some means (DHCP, (m)DNS, flash config) for the broker.